_config.yml

```diff
@@ -178,7 +178,7 @@ fetch-remote:
 
   - repo: "https://github.com/distribution/distribution"
     default_branch: "main"
-    ref: "release/2.7"
+    ref: "main"
```

Comment:
if anything this should be bumped to `2.8` 🤔

Reply:
Copy that, I've changed it to 2.8 now.

Reply:
Ideally, I'd love us to remove all these registry docs. We no longer own upstream. We should push CNCF to host registry docs. CC: @justincormack